overlord/snapstate/snapstatetest/devicectx.go

```diff
@@ -57,6 +62,11 @@ func MockDeviceModel(model *asserts.Model) (restore func()) {
 	return MockDeviceContext(deviceCtx)
 }
 
+func MockOperatingMode(operatingMode string) (restore func()) {
```

Comment:
this doesn't seem very useful alone, I would expect one that takes both a model and an operatingMode